hera_mc/geo_handling.py

```diff
@@ -334,7 +335,7 @@ def plot_stations(self, stations_to_plot_list, query_date, **kwargs):
         query_date:  date to use to check if active
         kwargs:  arguments for marker_color, marker_shape, marker_size, show_label, xgraph, ygraph
         """
-        import matplotlib.pyplot as plt
+        # import matplotlib.pyplot as plt
```

Comment:
Why was this commented out? By my reading it breaks the code if `self.testing` is False.

Reply:
oops, that's a mistake, I took it out because of a weird thing I was seeing but I thought I'd put it back in. Sorry, I'll fix it.

Reply:
I've got it all pulled up right now so I went ahead and made that change myself.

Reply:
thanks!